docs/pages/access-controls/sso/github-sso.mdx

```diff
@@ -32,86 +32,110 @@ https://PROXY_ADDRESS/v1/webapi/github/
 address of the Teleport Proxy Service</ScopedBlock><ScopedBlock
 scope="cloud">your Teleport Cloud tenant address</ScopedBlock>.
 
-Instructions for creating a GitHub OAuth app are available here:
+Instructions for creating a GitHub OAuth app are available in [GitHub's
+documentation](https://docs.github.com/en/developers/apps/building-oauth-apps/creating-an-oauth-app)
 
-[Creating an OAuth App](https://docs.github.com/en/developers/apps/building-oauth-apps/creating-an-oauth-app)
+Create a client secret to use along with the client ID in the next step.
 
 ## Step 2/3. Create a GitHub authentication connector
 
-Define a GitHub authentication connector by creating a file called `github.yaml`
-with the following content:
+Define a GitHub authentication connector using `tctl`. Update this example
+command with your OAuth app's client ID, client secret, and the roles you want
+to map from your GitHub organization to Teleport roles. See [tctl sso configure
+github](../../reference/cli.mdx#tctl-sso-configure-github) for a full reference
+of flags for this command:
 
-<Tabs>
-<TabItem label="Open Source" scope="oss">
+```code
+$ tctl sso configure github \
+--id=<Var name="GITHUB-CLIENT-ID"/>\
+--secret=<Var name="GITHUB-CLIENT-SECRET"/> \
+--teams-to-roles=<Var name="ORG-NAME,GITHUB-TEAM,access,editor"/> \
+> github.yaml
+```
+
+The contents of `github.yaml` should resemble the following:
 
 ```yaml
 kind: github
-version: v3
 metadata:
-  # Connector name that will be used with `tsh --auth=github login`
   name: github
 spec:
-  # Client ID of your GitHub OAuth App
+  api_endpoint_url: ""
   client_id: <client-id>
-  # Client secret of your GitHub OAuth App
   client_secret: <client-secret>
-  # Connector display name that will be shown on the Web UI login screen
   display: GitHub
-  # Callback URL that will be called after successful authentication
+  endpoint_url: ""
   redirect_url: https://<proxy-address>/v1/webapi/github/callback
-  # Mapping of org/team memberships onto allowed roles
+  teams_to_logins: null
   teams_to_roles:
-    - organization: octocats # GitHub organization name
-      team: admins # GitHub team name within that organization
-      # Maps octocats/admins to the "access" Teleport role
+    - organization: ORG-NAME
       roles:
         - access
+        - editor
+      team: GITHUB-TEAM
```

Comment:
It isn't immediately clear how can you utilize multiple team-to-role mappings, which will be a common use case. Do you mind adding a `<Details>` section covering that?

I think the easiest way to do so is via multiple instances of `--teams-to-roles` flag, like so:

```
> tctl sso configure github --id GITHUB_CLIENT_ID --secret GITHUB_SECRET -r octocats,devs,access,auditor,editor -r foo,bar,access

kind: github
metadata:
  name: github
spec:
  api_endpoint_url: ""
  client_id: GITHUB_CLIENT_ID
  client_secret: GITHUB_SECRET
  display: ""
  endpoint_url: ""
  redirect_url: https://boson.tener.io:3080/v1/webapi/github/callback
  teams_to_logins: null
  teams_to_roles:
  - organization: octocats
    roles:
    - access
    - auditor
    - editor
    team: devs
  - organization: foo
    roles:
    - access
    team: bar
version: v3
```

If someone wants to edit the YAML file manually, they can also pick up the syntax from the example above.

Reply:
Resolved in b2cd72ce7c2dc70e1c2df62107ef396d1a6d26a7